src/xbot/common/subsystems/pose/commands/SetRobotHeadingCommand.java

```diff
@@ -0,0 +1,35 @@
+package xbot.common.subsystems.pose.commands;
+
+import com.google.inject.Inject;
+
+import xbot.common.command.BaseCommand;
+import xbot.common.subsystems.pose.BasePoseSubsystem;
+
+public class SetRobotHeadingCommand extends BaseCommand {
+
+    BasePoseSubsystem poseSubsystem;
+    double heading;
+    
+    @Inject
+    public SetRobotHeadingCommand(BasePoseSubsystem poseSubsystem) {
+        this.poseSubsystem = poseSubsystem;
+        heading = BasePoseSubsystem.FACING_AWAY_FROM_DRIVERS;
+    }
+    
+    public void setHeadingToApply(double heading) {
```

Comment:
I guess we would mostly only ever do this with 90 so no need to support a property based version as well? I suppose we could muck with the currentHeading directly if need be..